README.md

```diff
@@ -64,6 +64,7 @@ Current release info
 | Name | Downloads | Version | Platforms |
 | --- | --- | --- | --- |
 | [![Conda Recipe](https://img.shields.io/badge/recipe-rubrix-green.svg)](https://anaconda.org/conda-forge/rubrix) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/rubrix.svg)](https://anaconda.org/conda-forge/rubrix) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/rubrix.svg)](https://anaconda.org/conda-forge/rubrix) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/rubrix.svg)](https://anaconda.org/conda-forge/rubrix) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-rubrix--client-green.svg)](https://anaconda.org/conda-forge/rubrix-client) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/rubrix-client.svg)](https://anaconda.org/conda-forge/rubrix-client) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/rubrix-client.svg)](https://anaconda.org/conda-forge/rubrix-client) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/rubrix-client.svg)](https://anaconda.org/conda-forge/rubrix-client) |
```

Comment:
Not sure if these links have to be updated since they mention a lot of times "rubrix-client".

Reply:
The readme is auto generated based on what is in the recipe. Since, I have dropped the `rubrix-client` altogether, I would not expect to see it in the readme.